src/openrct2-ui/windows/Research.cpp

```diff
@@ -362,16 +363,33 @@ void window_research_development_page_paint(rct_window* w, rct_drawpixelinfo* dp
     else
     {
         // Research type
-        stringId = STR_RESEARCH_UNKNOWN;
+        rct_string_id strings[2] = { STR_RESEARCH_UNKNOWN, 0 };
```

Comment:
```suggestion
        std::array<rct_string_id, 2> strings = { STR_RESEARCH_UNKNOWN, 0 };
```
If it works, prefer the `std::array`